### DISCUSSION TITLE
2023-12-27 Weekly Meeting 139 Notes/Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#433] [Weekly Meetings]
Not an official meeting. Casual discussion or working session for comms channels.

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on building the metaverse, whatever that means to you, you're welcome here (T&Cs and Code of Conducts apply).

(was: bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.)

We are concerned about practical standards, and helping grassroots members. We have a seat at MSF and are part of the W3C by charter. We intend to serve as a social connection between the different organisations, as a less formal organisation.

If you’re an observer or here for the first time, this is a general community meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We're glad you're here. What do you like doing?

We welcome you to introduce yourself in the 🔈 omi-weekly-meeting chat as we dive into the agenda items.

### Agenda
1. Backlog re-activation - done! Tasks: https://github.com/orgs/omigroup/projects/7/views/2
2. OMI wiki - can be integrated into github, according to mtv969 https://discord.com/channels/770382203782692945/880136862952489000/1184589126125760533
3. "map of open metaverse components?" - "Is there a "map the open metaverse" thing anywhere? By this I do NOT mean virtual worlds, it may have a small category listing those - e.g. Overte, V-Sekai, but I mean the protocols and tools that can help build the metaverse, from a FOSS perspective. So Blender, Inkscape, Godot, the glTF specs, and then the more behind-the-scenes stuff, like networking infrastructure, cloud stuff (Kubernetes?) ... stuff like that. "
Is this something that we can do as a OMF initiative, pitch to TLF?
4. Question for the group 
5. Demos and exploration!

### Working group updates (https://github.com/orgs/omigroup/teams)
none this week

Past meeting: https://hackmd.io/@XR/omigroup/https%3A%2F%2Fhackmd.io%2F%40XR%2Fomi_10-12-23 
and https://github.com/omigroup/omigroup/discussions/431

**Comment by indiebio** [comment]
demo shown in chat:
Pearl Hyacinth: https://newart.city/show/foxp2

**Comment by indiebio** [comment]
Demo shown in call by codefrau:
https://github.com/codefrau/jasmine

**Comment by indiebio** [comment]
antpb showed a demo that should be ready to launch in next weeks.

**Reply by mrmetaverse** [reply]
https://3ov.xyz/
